From e7bb1508373b22cd88e1ae0c0c5a9a548f1dfbb8 Mon Sep 17 00:00:00 2001
From: Jean-Baptiste Kempf <jb@videolan.org>
Date: Sun, 1 Mar 2015 18:30:10 +0100
Subject: [PATCH] Avparser: fix compilation with some older libav/FFmpeg
 versions

Ref #12135
---
 modules/packetizer/avparser.c | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/modules/packetizer/avparser.c b/modules/packetizer/avparser.c
index c5d2891..fdc4573 100644
--- a/modules/packetizer/avparser.c
+++ b/modules/packetizer/avparser.c
@@ -38,6 +38,20 @@
 #include "../codec/avcodec/avcommon.h"
 
 #include "avparser.h"
+
+#if !LIBAVCODEC_VERSION_CHECK( 55, 52, 0, 63, 100 )
+#include <libavutil/mem.h>
+static inline void avcodec_free_context( AVCodecContext **ctx )
+{
+    if( !*ctx )
+        return;
+
+    av_free( (*ctx)->extradata );
+    av_free( *ctx );
+    *ctx = NULL;
+}
+#endif
+
 /*****************************************************************************
  * Module descriptor
  *****************************************************************************/
-- 
2.3.3