--- a/modules/audio_output/oss.c.org	2015-05-15 17:49:47.923275755 +0300
+++ b/modules/audio_output/oss.c	2015-05-15 17:56:19.189384256 +0300
@@ -33,11 +33,7 @@
 #include <sys/types.h>
 #include <fcntl.h>
 #include <sys/ioctl.h>
-#ifdef HAVE_SOUNDCARD_H
-# include <soundcard.h>
-#else
-# include <sys/soundcard.h>
-#endif
+#include "/usr/lib/oss/include/sys/soundcard.h"
 
 #ifndef SNDCTL_DSP_HALT
 # define SNDCTL_DSP_HALT SNDCTL_DSP_RESET
